src/lighteval/metrics/metrics_sample.py

```diff
@@ -566,7 +566,7 @@ def __init__(
         self.normalize_gold = normalize_gold
         self.normalize_pred = normalize_pred
 
-    def compute(self, golds: list[str], predictions: list[str]) -> dict:
```

Comment:
thanks for this fix!

Reply:
feel free to fix the other functions with a similar issue

Reply:
Thanks! I checked the file and seems like BLEURT is the only other metric without `**kwargs` and experiencing the same formatted_doc input error. When I added it, there were several other issues so I will create a separate PR for it :).